From b95853bd4839e5f9f0aa6afee373f8b5a6d3be11 Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Thu, 15 Sep 2016 13:31:53 -0700
Subject: [PATCH] TOOLS-1546 cr.joyent.us "new user guide" link overlaps with
 header on change screen Reviewed by: Alex Wilson <alex.wilson@joyent.com>
 Approved by: Alex Wilson <alex.wilson@joyent.com>

---
 images/appserver/theme/GerritSite.css | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/images/appserver/theme/GerritSite.css b/images/appserver/theme/GerritSite.css
index 63dfb71..6be053c 100644
--- a/images/appserver/theme/GerritSite.css
+++ b/images/appserver/theme/GerritSite.css
@@ -2,10 +2,10 @@
  * See the Gerrit documentation for information about this file.
  */
 table.crjHeader {
-	width: 100%;
 	position: fixed;
 	right: 5px;
-	top: 35px;
+	top: 27px;
+	width: 300px;
 
 	font-weight: bold;
 	text-align: right;
-- 
2.21.0

